dist-git/dist-git.spec

```diff
@@ -59,16 +59,10 @@ package sources.
 Summary:        SELinux support for dist-git
 
 BuildRequires:  checkpolicy
-BuildRequires:  policycoreutils
-BuildRequires:  selinux-policy-devel
 BuildRequires:  hardlink
 
 Requires:       %name = %version-%release
-%if "%{_selinux_policy_version}" != ""
-Requires:       selinux-policy >= %{_selinux_policy_version}
-%endif
-Requires(post):   /usr/sbin/semodule, /sbin/restorecon
-Requires(postun): /usr/sbin/semodule, /sbin/restorecon
+%{?selinux_requires}
```

Comment:
It looks this macro is not yet finished :-( the %selinux_requires is not defined by redhat-rpm-config nor any other package in minimal buildroot. I [reported this problem](https://bugzilla.redhat.com/show_bug.cgi?id=2311871).

The docs are not updated: https://fedoraproject.org/wiki/SELinux/IndependentPolicy#Example_spec_file_changes_to_incorporate_-selinux_subpackage

IOW, we need to do:
BuildRequires: selinux-policy-devel
%{?selinux_requires}